### PR TITLE
Redis cluster ssl

### PR DESCRIPTION
**Pull request by bharadwajrembar**
Fixes #972 , the password is not propagated to the JedisClusterClient and thus, throws an error stating `NOAUTH` when trying to connect to a Redis instance with Authentication enabled. This PR fixes that. Please review. @kishorebanala @apanicker-nflx , cc: @skyrocknroll

### CODE REVIEW

File: redis-persistence/src/main/java/com/netflix/conductor/jedis/RedisClusterJedisProvider.java [hunk 1]
```diff
@@ -27,9 +27,15 @@ public RedisClusterJedisProvider(HostSupplier hostSupplier){
     public JedisCommands get() {
         // FIXME This doesn't seem very safe, but is how it was in the code this was moved from.
         Host host = new ArrayList<Host>(hostSupplier.getHosts()).get(0);
+        String password = host.getPassword();
         GenericObjectPoolConfig poolConfig = new GenericObjectPoolConfig();
         poolConfig.setMinIdle(5);
         poolConfig.setMaxTotal(1000);
-        return new JedisCluster(new HostAndPort(host.getHostName(), host.getPort()), poolConfig);
+        if (password.length() == 0) {
```
Comment (kishorebanala) [inline redis-persistence/src/main/java/com/netflix/conductor/jedis/RedisClusterJedisProvider.java]: Consider using `StringUtils.isNotBlank` to null check.

**Comment by kishorebanala** [comment]
Hey @bharadwajrembar, are you still planning to work on this feature? We still feel this would be an important addition to ES modules.

**Comment by bharadwajrembar** [comment]
@kishorebanala I am held up at the moment. I would not be able to take on this as well as https://github.com/Netflix/conductor/pull/1209 as of now. 

If I find bandwidth, I will surely pick it up and inform you of the same.

**Comment by apanicker-nflx** [comment]
Closing due to inactivity. Please reopen if you plan to resume working on this.

**Pull request closed without merging**